decred/decred/dcr/nets.py

```diff
@@ -6,21 +6,16 @@
 from . import mainnet, testnet, simnet
 
 
-mainnet = mainnet
-testnet = testnet
-simnet = simnet
+the_nets = {n.Name: n for n in (mainnet, testnet, simnet)}
```

Comment:
You can already use something like:
```
from decred.dcr import nets; (nets.mainnet, nets.testnet, nets.simnet)
```
to refer to a net parameters module.

The `decred.dcr.nets.parse` function (and the similar `decred.config.tinyNetConfig` one) get passed the `somenet.Name` parameter, apparently needed because of the `testnet` vs. `testnet3` difference (`tinyNetConfig` takes that into account).

Not sure what the best approach is here.

Regardless of what we end up with, I would create a `decred.dcr.nets` package (directory) and put the three network parameter modules in there.

Reply:
Huh. Didn't know you could do that. 

**nets** package sounds good. Follow up?

Reply:
Yep, I'll make a separate PR for that.